src/previewFrame/elementHighlighterScript.js

```diff
@@ -0,0 +1,36 @@
+const elementHighlighterScript = `(${function() {
+  window.addEventListener('message', (message) => {
+    const data = JSON.parse(message.data);
+    const {selector: {selector: selectorFromMessage}} = data;
+    const selector = selectorFromMessage;
+    updateCovers(selector);
+  });
+
+  function updateCovers(selector) {
+    const highlighterElements =
+    window.document.querySelectorAll('.popcode_HIGHLIGHTER');
+    if (highlighterElements.length !== 0) {
+      highlighterElements.forEach((highlighterElement) => {
```

Comment:
I prefer `for (const highlighterElement of highlighterElements)` in modern JS

Reply:
updated